packages/lodestar/src/executionEngine/index.ts

```diff
@@ -35,3 +42,11 @@ export function initializeExecutionEngine(opts: ExecutionEngineOpts, signal: Abo
       return new ExecutionEngineHttp(opts, signal);
   }
 }
+
+export function initializeExecutionBuilder(opts: ExecutionBuilderOpts, signal: AbortSignal): IExecutionBuilder {
+  switch (opts.mode) {
+    case "http":
+    default:
+      return new ExecutionBuilderHttp(opts, signal);
+  }
+}
```

Comment:
Should this be moved to `packages/lodestar/src/executionBuilder/index.ts` while re-using code from `packages/lodestar/src/executionEngine/*` if necessary?

Reply:
yes makes sense, will move :slightly_smiling_face: